share/spice/cricket/cricket.css

```diff
@@ -0,0 +1,136 @@
+.zci--cricket .card {
+    background: #FFFFFF;
+    font-size: 16px;
+    box-shadow: 0 10px 20px rgba(0, 0, 0, 0.19), 0 6px 6px rgba(0, 0, 0, 0.23);
+    box-sizing: border-box;
+}
+
+.zci-cricket-content {
+    color: #888;
+    padding: 20px;
+    border-top: 2px solid #F7F7F7;
+    border-bottom: 2px solid #F7F7F7;
+}
+
+.zci-cricket-content > span:first-child {
+    float: left;
```

Comment:
we have a `pull--left` class, likewise `pull--right` that apply left/right floats